cql-engine-fhir/src/main/java/org/opencds/cqf/cql/model/HL7FhirModelResolver.java

```diff
@@ -6,7 +6,6 @@
 import java.util.Calendar;
 
 import org.hl7.fhir.dstu2.model.*;
```

Comment:
This should be using `import org.hl7.fhir.instance.model.*;`

Reply:
I respect your suggestion and finds it logical. But release 4.0.0 says (https://hapifhir.io/hapi-fhir/docs/introduction/changelog.html#hapi-fhir-400-igloo) : "Breaking Change: The HL7.org DSTU2 structures (and ONLY the HL7.org DSTU2 structures) have been moved to a new package. Where they were previously found in org.hl7.fhir.instance.model they are now found in org.hl7.fhir.dstu2.model. ..."
Please reconsider!

Reply:
I see... Then we probably won't need the both the Dstu2FhirModelResolver and HL7FhirModelResolver as they seem redundant. Thoughts @JPercival ?

Reply:
I think you're right. I think we can remove the HL7FhirModelResolver.

Reply:
Removed HL7FhirModelResolver